sdk/eventhub/azure-eventhub/azure/eventhub/_constants.py

```diff
@@ -44,6 +44,8 @@
 MGMT_STATUS_CODE = b"status-code"
 MGMT_STATUS_DESC = b"status-description"
 USER_AGENT_PREFIX = "azsdk-python-eventhubs"
+UAMQP_LIBRARY = "uamqp"
+PYAMQP_LIBRARY = "pyamqp"
```

Comment:
For a user-agent, both these should be paired with their version:
`uamqp_moniker = f"uamqp/{uamqp.__version__}"`